src/api/dto/add-api-rating.dto.ts

```diff
@@ -1,13 +1,18 @@
-import { ApiProperty } from "@nestjs/swagger";
-import { IsNotEmpty, IsString, Max, Min } from "class-validator";
+import { ApiProperty, ApiPropertyOptional } from "@nestjs/swagger";
+import { IsNotEmpty, IsOptional, IsString, Max, Min } from "class-validator";
 export class ApiRatingDto {
-    @ApiProperty()
+    @ApiPropertyOptional()
     @IsString()
-    review: string
+    review? : string
```

Comment:
will IsString() allow for null values.. did u chekc??

Reply:
It dosent. Fixed it